Doberman/Node.py

```diff
@@ -1,14 +1,19 @@
 import Doberman
 import requests
 import itertools
-import numpy as np
+try:
+    import numpy as np
+except ImportError:
+    np = None
```

Comment:
Is this purely for the single `numpy.log10`? It is also available in the standard library, in `math`

Reply:
Good point